src/modules/map/components/openlayers/OpenLayersPopover.vue

```diff
@@ -79,7 +79,7 @@ export default {
             this.$emit('close')
         },
         printContent() {
-            promptUserToPrintHtmlContent(this.$refs.mapPopoverContent.outerHTML)
+            promptUserToPrintHtmlContent('mapPopoverContent')
```

Comment:
I would maybe also change https://github.com/geoadmin/web-mapviewer/blob/develop/src/utils/print.js
at least update the JSDoc so that it is clear we can also give a HTML ID here (and not necessarily a HTMLElement as stated in the doc)